wechatpay/README.md

```diff
@@ -0,0 +1,55 @@
+# Stripe Android SDK WeChatPay module
+Module to provide support for `PaymentIntent` with `wechat_pay` as its payment method.
+ 
+# Overview
+* This module will provide a `PaymentAuthenticator` implementation to authenticate a [PaymentIntent](https://stripe.com/docs/api/payment_intents) with [next_action.wechat_pay_redirect_to_android_app](https://stripe.com/docs/api/payment_intents/object#payment_intent_object-next_action-wechat_pay_redirect_to_android_app) by invoking the WeChat Pay SDK and pass the required parameters from that hash.
+
+* No extra code is needed when using this module, however you will need to declare dependency on the actual WeChat Pay SDK, as this module uses reflection to invoke it.
+
+# Usage
+* In app/build.gradle, add these dependencies:
+    ```gradle
+    dependencies {
+      // Stripe dependencies, make sure they have the same $stripe_sdk_version
+      implementation 'com.stripe:stripe-android:$stripe_sdk_version' // Main Stripe SDK
+      implementation 'com.stripe:stripe-wechatpay:$stripe_sdk_version' // WeChat Pay module
+      // WeChat Pay SDK, make sure 6.7.0 is used
+      implementation 'com.tencent.mm.opensdk:wechat-sdk-android-without-mta:6.7.0'
+    }
+    ```
+
+* Then in your app, just confirm and capture the result of the `PaymentIntent` with WeChat Pay like others(see an complete example [here](https://stripe.com/docs/payments/integration-builder)), WeChat app on the phone will be opened to confirm the payment.
+    ```kotlin
+    class CheckoutActivity : AppCompatActivity() {
+        private fun startCheckout() {
+            payButton.setOnClickListener {
```

Comment:
Being already familiar with the SDk I know the type of cardInputWidget, but someone new to the SDK might not be familiar. 

It might be helpful to include a simple xml, or a comment describing the variables.  In (this)[https://stripe.com/docs/payments/accept-a-payment?platform=android] example we show them as member variables in the class and then have a comment that instructs them to instantiate it.

Reply:
trying to just focus on wechat specific context here, removed the unrelated part